ROADMAP-v1.md

```diff
@@ -0,0 +1,36 @@
+# ROADMAP to version 1
+
+* Update to 2024.1 specification; that includes:
+  * [Remove the JSON Standard Primitive type](https://github.com/LionWeb-io/lionweb-typescript/issues/161)
+  * [Add support for value types](https://github.com/LionWeb-io/lionweb-typescript/issues/160)
+
+* Re-organize into packages (this is [issue #154](https://github.com/LionWeb-io/lionweb-typescript/issues/154), which might make [issue #86](https://github.com/LionWeb-io/lionweb-typescript/issues/86) obsolete):
+  * `serialization`: all serialization types — issues [#107](https://github.com/LionWeb-io/lionweb-typescript/issues/107) and [#106](https://github.com/LionWeb-io/lionweb-typescript/issues/106) can be considered to address this as well
+  * `base`: base types such as `Node`
+  * `internal-utils`: internal utilities, such as for working with arrays, maps, doing topological sorting, etc.
+  * `m3`: LionCore M3
+  * `m2`: LionCore builtins, language (de-)serialization, factories, (maybe some!) convenience/helper functionality
+  * `m1`: model (de-)serialization
+  * `validation`: validation of serialization chunks — as standalone as possible
+  * `utilities`: ?
+  * `queries`: ?
+
+* Rename `InstantiationFacade` &rarr; `Factory`
+  * And `ExtractionFacade` &rarr; `Reflector`?
+
+* Give deserialization a better API:
+  * Pair up languages and their factories
+  * Make deserialization more configurable:
```

Comment:
I favor of this, but we should make sure that the usage should remain simple in case you don't need all those configurable options.

Reply:
Added an extra bullet point about that.